dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/EntityDefiner.java

```diff
@@ -90,7 +92,7 @@ else if ((id == null || id.length() != 2) && name.endsWith("ID"))
         }
 
         if (id == null)
-            throw new IllegalArgumentException(entityClass + " lacks public field with @Id or of the form *ID");
+            throw new MappingException(entityClass + " lacks public field with @Id or of the form *ID"); // TODO
```

Comment:
Is the TODO here to translate this message in the future?

Reply:
yes